src/test_workflow/benchmark_test/benchmark_test_cluster.py

```diff
@@ -141,7 +141,9 @@ def setup_cdk_params(self, config: dict) -> dict:
             "mlNodeCount": self.args.ml_node_count,
             "dataNodeStorage": self.args.data_node_storage,
             "mlNodeStorage": self.args.ml_node_storage,
-            "jvmSysProps": self.args.jvm_sys_props
+            "jvmSysProps": self.args.jvm_sys_props,
+            "use50PercentHeap": str(self.args.use_50_percent_heap).lower(),
```

Comment:
This does not seem to be used anywhere?

Reply:
Just realize this is just a setting for part of the clustercdk then I have no issues.